after fix travis builds :+1: 

``` bash
partner_firstname/tests/base.py:74:80: E501 line too long (81 > 79 characters)
partner_firstname/tests/test_create.py:66: [W0101(unreachable), CompanyCase.setUp] Unreachable code
```

--- a/partner_firstname/tests/base.py
+++ b/partner_firstname/tests/base.py
@@ -71,7 +71,8 @@ class BaseCase(TransactionCase, MailInstalled):
     def test_copy(self):
         """Copy the partner and compare the result."""
         self.expect(self.lastname, u"%s (copy)" % self.firstname)
-        self.changed = self.original.with_context(copy=True, lang="en_US").copy()
+        self.changed = (self.original.with_context(copy=True, lang="en_US")
+                        .copy())
 
     def test_one_name(self):
         """Test what happens when only one name is given."""

--- a/partner_firstname/tests/test_create.py
+++ b/partner_firstname/tests/test_create.py
@@ -62,8 +62,9 @@ class CompanyCase(PersonCase):
     context = {"default_is_company": True}
 
     def setUp(self):
-        return super(CompanyCase, self).setUp()
-        self.values.update(lastname=self.values["name"], firstname=False)
+        super(CompanyCase, self).setUp()
+        self.good_values.update(lastname=self.values["name"], firstname=False)
+        self.values = self.good_values.copy()
 
 
 class UserCase(PersonCase, MailInstalled):

